types/ember__object/events.d.ts

```diff
@@ -1,34 +1,28 @@
-import { ObserverMethod } from '@ember/object/-private/types';
+import { ObserverMethod, Callback } from '@ember/object/-private/types';
 
 /**
  * Add an event listener
  */
-export function addListener<Context, Target>(
-    obj: Context,
-    key: keyof Context,
-    target: Target,
-    method: ObserverMethod<Target, Context>,
+export function addListener(
```

Comment:
the removal of these type params is a backwards-incompatible change that will break anyone who has things like

```ts
addListener<C,T>(...
```
in their app. We'd want to think very carefully before doing something like this